### PR TITLE
languages: re-generate the list of drivers

### PR DESCRIPTION
**Pull request by bzz**
Result of the `make languages`

### CODE REVIEW

File: languages.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
 		"Name": "Bash",
 		"Language": "bash",
 		"Status": "beta",
-		"SDKVersion": "2.16.0",
+		"SDKVersion": "2.16.4",
```
Comment (creachadair) [inline languages.json]: Oops, I guess we forgot to do this the last couple times. 😢

Comment (dennwc) [reply languages.json]: It's quite hard to update on each point release of SDK without automation, so we usually update only if new release improves the performance or fixes something for this driver.

Comment (creachadair) [reply languages.json]: Agreed—I wasn't complaining, but it's a good point in favour of finding a way to automate the check, at least (as we've discussed previously).

**Comment by bzz** [comment]
Yup. And thank you for the prompt reviews.

**Pull request merged**